niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/utils/PermissionUtils.java

```diff
@@ -92,4 +104,80 @@ public static void checkTester(BasePermissionTester tester){
         }
     }
 
+
+
+    /**
+     * Ensures that the passed in permissions are defined in manifest and attempts to grant a
```

Comment:
Is this comment still true? It looks like it just checks for the presence of the permission in the manifest, but I didn't see a path where it attempts to grant a dangerous permission.

Reply:
yes... I also think it's a false information.
will update

Reply:
Updated comment sounds good, thanks!